tests/vmi_configuration_test.go

```diff
@@ -2306,6 +2306,12 @@ var _ = Describe("Configurations", func() {
 			Expect(err).ToNot(HaveOccurred())
 			tests.WaitForSuccessfulVMIStart(vmi)
 
+			test_smbios := &cmdv1.SMBios{Family: "", Product: "", Manufacturer: ""}
```

Comment:
Do you want to do this as a pre-caution? I did not find the place in the code where they would have been set.

Reply:
These values are already present on some clusters. (e.g. those deployed via HCO)